src/react/hoc/graphql.tsx

```diff
@@ -5,13 +5,16 @@ import { withQuery } from './query-hoc';
 import { withMutation } from './mutation-hoc';
 import { withSubscription } from './subscription-hoc';
 import { OperationOption, DataProps, MutateProps } from './types';
+import { OperationVariables } from '../../core';
+
+type ChildProps<TData extends {}, TGraphQLVariables extends OperationVariables> =
```

Comment:
```suggestion
type ChildProps<TData extends Record<string, any>, TGraphQLVariables extends OperationVariables> =
```

See comment above on reasoning.

Reply:
As far as I can tell `extends object` and `extends Record<string, any>` are functionally equivalent. The docs recommend `object`, but I've opted for `Record<string, any>` across the board for consistency.